src/libpsl-native/src/geterrorcategory.cpp

```diff
@@ -0,0 +1,27 @@
+#include "geterrorcategory.h"
+
+#include <assert.h>
+#include <sys/types.h>
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+//! @brief Maps Linux errno to PowerShell ErrorCategory
```

Comment:
Neither name nor constant can change - it's a public api.
Simple to write is not necessarily more maintainable though, which is why I suggested copying the enum - making it even clearer what the origin of those constants are.